notebooks/monitoring.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+echo ==================================
```

Comment:
Could you move all the top-level `echo` calls into a `main` function? I'm thinking a structure like this:
```bash
#!/bin/bash

function get_first_thing () {

}

function get_second_thing () {

}

# More helpers...

function main() {
  echo ...
  echo ...
}

main
```